src/System.Runtime.WindowsRuntime/ref/System.Runtime.WindowsRuntime.csproj

```diff
@@ -23,10 +23,6 @@
     <Compile Include="System.Runtime.WindowsRuntime.Manual.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ReferencePath Include="$(ExternalAPIsPath)\NetFX\Contracts\DesignTimeFacades\mscorlib.dll" />
```

Comment:
I think roslyn is less aggressive about walking type refs (in some cases, more in others).